src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/table/table-performance-counter.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class TablePerformanceCounterService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  get(service_type: string, service_id: string) {
+    return this.http.get(`/api/perf_counters/${service_type}/${service_id}`)
```

Comment:
Not sure if the path should stay this way, because "/api/$type/$id/perf_counters" would look a lot better IMO.